.gitmodules

```diff
@@ -0,0 +1,4 @@
+[submodule "kaitai_struct_compiler"]
+	path = kaitai_struct_compiler
+	url = https://github.com/ManasviGoyal/kaitai_struct_compiler.git
+	branch = ManasviGoyal/kaitai_awkward_target
```

Comment:
I think it would be best to have a specific commit instead of a branch. So that this is always reproducible.
Then we manually push ahead the commit when needed.